### DISCUSSION TITLE
Farbige Plattenicons?

### DISCUSSION THREAD

**Discussion by NickRTR** [NickRTR-Schule/Gehweg-Parcours#22] [Umfragen]
Sollen wir unsere icons auf den Platten in farbige icons ändern oder den cleanen schwarz-weiß look bewahren?

**Comment by NickRTR** [comment]
@luk4xsx @Adrian-Cxll

**Reply by Adrian-Cxll** [reply]
Naja, wegen "Gehweg"..
Wenn man verschiedene Farben nehmen würde, wären Grautöne nicht passend?

**Reply by NickRTR** [reply]
Stimmt, die Flagge und Kacke aber farbig.

**Comment by Adrian-Cxll** [comment]
ye
